containers/Register/register.tsx

```diff
@@ -1,99 +1,209 @@
 import { useState } from "react";
-import { Formik, Form, useFormik } from 'formik';
-import Input from "@/components/Input";
-import Field from "@/components/Field";
-import ErrorMessage from "@/components/ErrorMessage";
-import { DISABLE_SPEEDY } from "styled-components/dist/constants";
-
-
-interface RegisterProps {
-
-}
+import { useFormik } from 'formik';
+import { useRouter } from 'next/navigation'
+import { get } from 'lodash'
+import "react-toastify/dist/ReactToastify.css";
 
-interface Error {
-  username: string;
+import { regex } from '@/constants'
+import { validate } from '@/helpers'
+import Field from "@/components/Field";
+import { ToastContainer, toast } from "react-toastify";
 
-}
+import {
+  useSignIn,
+  useSignUp
+} from '@/lib/hooks'
+import Button from "@/components/Button";
 
-interface FormPayload {
-  username: string;
-  password: string;
-  [key: string]: string;
-}
 
 interface FormField {
   fieldName: string;
-  dot: boolean;
   label: string;
-  disabled: boolean;
+  disabled?: boolean;
   type: string;
+  required: true;
+  regex?: RegExp;
+  regexMessage?: string;
 }
 
-const signInForms: FormField [] = [
+const signInForms: FormField[] = [
```

Comment:
remove these const from component.. add somewhere in constant filels or helper file